content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+e non viene aggiornato se il tag dell'immagine dovesse cambiare successivamente.
+
+Ad esempio, creando un Deployment con un'immagine il cui tag _non_ è 
+`:latest`, e successivamente aggiornando il tag di quell'immagine a `:latest`, il campo
+ `imagePullPolicy` _non_ cambierà su `Always`. 
+È necessario modificare manualmente la policy di pull di ogni oggetto dopo la sua creazione.
+{{< /note >}}
+
+Quando `imagePullPolicy` è definito senza un valore specifico, esso è impostato su `Always`.
+
+## Immagini Multi-architecture con indici
+
+Oltre a fornire immagini binarie, un _container registry_ può fornire un [indice per l'immagine di un container](https://github.com/opencontainers/image-spec/blob/master/image-index.md). 
+L'indice di un'immagine può puntare a più [file manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md) ciascuno per una versione specifica dell'architettura di un container. 
+L'idea è che si può avere un unico nome per una stessa immagine (ad esempio: `pause`, `example/mycontainer`, `kube-apiserver`) e permettere a diversi sistemi di recuperare l'immagine binaria corretta a seconda dell'architettura della macchina che la sta utilizzando.
+
+
+Kubernetes stesso tipicamente nomina le immagini dei container tramite il suffisso `-$(ARCH)`. 
+Per la garantire la retrocompatibilità è meglio generare le vecchie immagini con dei suffissi. 
+L'idea è quella di generare, ad esempio, l'immagine `pausa` che ha il manifest per tutte le versioni del processore _arch_ e
+ `pausa-amd64` che è retrocompatibile per le vecchie configurazioni o per quei file YAML in cui sono specificate le immagini con i suffissi.
+
+## Utilizzare un registro privato
+
+I registri privati possono richiedere l'utilizzo di chiavi per accedere alle immagini in essi contenute.  
+Le credenziali possono essere fornite in molti modi:
+  - configurando i nodi in modo tale da autenticarsi al registro provato
+    - tutti i pod possono acquisire informazioni da qualsiasi registro privato configurato
+    - è necessario che l'amministratore del cluster configuri i nodi in tal senso
+  - tramite immagini prescaricate
+    - tutti i pod possono accedere alle immagini salvate sulla cache del nodo a cui si riferiscono
+    - è necessario effettuare l'accesso come root di sistema su ogni nodo per inserire questa impostazione
+  - specificando _ImagePullSecrets_ su un determinato pod
+    - solo i pod che forniscono le proprie chiavi hanno la possibilità di accedere al registro privato
+  - tramite estensioni locali o specifiche di un _Vendor_
+    - se si sta utilizzando una configurazione personalizzata del nodo oppure se manualmente, o tramite il _cloud provider_,
+      si implementa un meccanismo di autenticazione del nodo presso il _container registry_.
+
+Di seguito la spiegazione dettagliata di queste opzioni.
+
+### Configurazione dei nodi per l'autenticazione ad un registro privato
```

Comment:
```suggestion
### Configurazione dei nodi per l'autenticazione ad un private registry
```

I think if the desired is "private registry" instead of "registro privato" as suggested by @fabriziopandini, we have to edit this line too.